src/lib/math/bigint/bigint.h

```diff
@@ -645,11 +680,58 @@ class BOTAN_PUBLIC_API(2, 0) BigInt final {
        */
       void randomize(RandomNumberGenerator& rng, size_t bitsize, bool set_high_bit = true);
 
+      /**
+       * Serialize the absolute value of this BigInt as a big endian
+       * encoding.
+       *
+       * If out is smaller than the total bytes of the BigInt then
+       * an exception is thrown.
+       *
+       * If out is larger than the total bytes of the BigInt then the
+       * necessary number of zeros are prefixed to produce the desired
+       * output length
+       *
+       * Zero-padding the binary encoding is useful to ensure that other
+       * applications correctly parse the encoded value as "positive integer",
+       * as a leading 1-bit may be interpreted as a sign bit. It also is
+       * necessary when using a fixed size encoding for the integers.
+       *
+       * @param out destination byte span for the integer value
+       */
```

Comment:
Perhaps mention that this aims to serialize according to IEEE 1363?

Reply:
I think at this point IEEE 1363 is completely dead to the point of referencing it causing more confusion than benefit; IEEE officially withdrew almost all of 1363 in 2019 and long before then it was obscure/obsolete. A great deal of terminology in the library (eg EMSA/EME) comes from IEEE 1363, and using it was definitely a bad bet on my part.